### DISCUSSION TITLE
Documentation improvements

### DISCUSSION THREAD

**Discussion by maxkratt** [FirstGearGames/FishNet#790] [Documentation]
Specify on [this page](https://fish-networking.gitbook.io/docs/manual/guides/technical-limitations#nested-networkobjects-and-networkbehaviours) that network object prefabs nested under another network object prefab cannot be unparented.

Add support for a PDF download of the API Documentation as supported here: https://dotnet.github.io/docfx/docs/pdf.html

List prominent up to date tutorials somewhere on the docs such as this one: https://www.youtube.com/watch?v=F5VFEy1_THw

Update the broken Master Server Toolkit link here: https://fish-networking.gitbook.io/docs/manual/guides/add-ons 
to their asset store page: https://assetstore.unity.com/packages/tools/network/master-server-toolkit-194832

Remove the Network Level of Detail section here: https://fish-networking.gitbook.io/docs/master/pro-learning-resources-and-support

Add the [CanoeWebRTC](https://github.com/gmrodriguez124/CanoeWebRTC) transport to the [transports page](https://fish-networking.gitbook.io/docs/manual/general/transports)

Mention [here](https://fish-networking.gitbook.io/docs/manual/guides/technical-limitations#nested-networkobjects-and-networkbehaviours) that predicted spawning cannot spawn an object nested to another.

`RollbackManager.PhysicsType.ThreeDimensional` should be changed to `RollbackPhysicsType.Physics` on this page: https://fish-networking.gitbook.io/docs/manual/guides/lag-compensation/raycast

The first code block [here](https://fish-networking.gitbook.io/docs/manual/guides/synchronizing/syncvar) should show the class inheriting from NetworkBehaviour to avoid confusing people.

**Comment by FirstGearGames** [comment]
Resolved all of these, thanks as always max!
